src/test/java/com/qualimente/training/addressbook/AddressTest.java

```diff
@@ -49,7 +49,7 @@ public void constructor_stores_arguments() {
 
   @Test
   public void constructor_permits_null_id(){
```

Comment:
why?

I really do want to verify that id is not required to create an Address object because ids are generated server-side